frontend/app/src/main/res/values/strings.xml

```diff
@@ -126,6 +126,24 @@
     <string name="achievement2">Deutscher Meister Senior Trail</string>
     <string name="achievement3">Deutscher Meister Senior Horsemanship</string>
     <string name="achievement4">Deutscher Meister Senior Showmanship</string>
+    <!-- Strings related to login -->
+    <string name="prompt_email">Email</string>
+    <string name="prompt_password">Password</string>
+    <string name="action_sign_in">Sign in or register</string>
+    <string name="action_sign_in_short">Sign in</string>
+    <string name="welcome">"Welcome!"</string>
+    <string name="invalid_username">Not a valid username</string>
+    <string name="invalid_password">Password must be >5 characters</string>
+    <string name="login_failed">"Login failed"</string>
+
+
+    <string-array name="roles">
+        <item>Nutzer</item>
+        <item>Admin</item>
+        <item>Trainer</item>
+    </string-array>
+    <!-- TODO: Remove or change this placeholder text -->
```

Comment:
Kannst du das bitte auch machen? :D Zeile darunter ist gemeint ;) und dann kannst du gleich auch noch die doppelten Leerzeichen in 147/148 entfernen